types/porep_proof_partitions.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// PoRepProofPartitions TODO(laser)
 type PoRepProofPartitions uint64
 
 const (
+	// TestPoRepProofPartitions TODO(laser)
 	TestPoRepProofPartitions = PoRepProofPartitions(iota)
+	// TwoPoRepPartitions TODO(laser)
```

Comment:
TwoPoRepPartitions indicates that two partitions were used to create a PoRep proof.